src/changelog/utils.py

```diff
@@ -138,11 +140,11 @@ def crunch_lines(self, line_list):
                 i += 1
         return line_list
 
-    def bump_version(self, version, release_type):
+    def bump_version(self, version: str, release_type: str) -> str:
         """
         Bumps a version number based on release_type
         """
-        x, y, z = version.split(".")
+        x, y, z = [int(i) for i in version.split(".")]
```

Comment:
if we are going to cast to int in the comprehension, there is no need for the cast in the if statement

Reply:
Yup. Whoops. Much cleaner now. 👍